src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/repository/BitbucketCloudTeam.java

```diff
@@ -23,52 +23,16 @@
  */
 package com.cloudbees.jenkins.plugins.bitbucket.client.repository;
 
-import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketHref;
+import com.cloudbees.jenkins.plugins.bitbucket.api.AbstractBitbucketTeam;
 import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketTeam;
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import java.util.List;
-import java.util.Map;
 
-public class BitbucketCloudTeam implements BitbucketTeam {
+public class BitbucketCloudTeam extends AbstractBitbucketTeam implements BitbucketTeam {
```

Comment:
Redundant as AbstractBitbucketTeam implements BitbucketTeam
```suggestion
public class BitbucketCloudTeam extends AbstractBitbucketTeam {
```

Reply:
fixed